app/actions/actions.js

```diff
@@ -1,12 +1,14 @@
 import { ADD_PRODUCER_DATA, ADD_CONSUMER_DATA } from '../constants/actionTypes';
 
+// Action creator that describes and generates the actions to be dispatched for Producer
 export const addProducerData = (newData) => ({
   type: ADD_PRODUCER_DATA,
   payload: {
     data: newData,
   },
 });
 
+// Action creator that describes and generates the actions to be dispatched for Producer
```

Comment:
same comment as above^
but also, the last word for this comment should be "Consumer"
also on that note, no need to capitalize producer/consumer, this applies for the comment above as well